index.html

```diff
@@ -37,33 +43,262 @@ <h3>
       <ul class="social">
         <li>
           <a href="https://twitter.com/ankit26k">
-            <img class="icon" src="/images/twitter.png" alt="twitter"
+            <img class="icon" src="images/twitter.png" alt="twitter"
           /></a>
         </li>
 
         <li>
           <a href="https://www.linkedin.com/in/ankit26k/">
-            <img class="icon" src="/images/Linkedin.png" alt="linkedin"
+            <img class="icon" src="images/Linkedin.png" alt="linkedin"
           /></a>
         </li>
 
         <li>
           <a href="https://twitter.com/ankit26k">
-            <img class="icon meta" src="/images/Vector1.png" alt="meta"
+            <img class="icon meta" src="images/Vector1.png" alt="meta"
           /></a>
         </li>
 
         <li>
           <a href="https://github.com/ankitt26">
-            <img class="icon" src="/images/vector2.png" alt="github"
+            <img class="icon" src="images/vector2.png" alt="github"
           /></a>
         </li>
 
         <li>
           <a href="https://www.instagram.com/ankit16t/">
-            <img class="icon" src="/images/vector3.png" alt="twofin"
+            <img class="icon" src="images/vector3.png" alt="twofin"
+          /></a>
+        </li>
+      </ul>
+    </section>
+
+    <!-- Work section -->
+    <section>
+      <div class="works">
```

Comment:
[OPTIONAL] I suggest you ad the class directly to the ``section`` tag and avoid the ``div``, the rule of thumb is that if you can avoid the ``div`` avoid 😄

Reply:
okay thanks 😀